src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
 export interface Figure {
+  shape:string,
+  color:string,
+  getArea: () =>{}
+}
 
+function roundedDownToHundredths(area:number):number {
+  return Math.floor(area * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape = 'triangle';
 
-}
+  constructor(
+    public color:string,
+    public a:number,
+    public b:number,
+    public c:number,
+  ) {
+    if (!this.isValidSides()) {
+      throw new Error('incorrect sides of triangle');
+    }
+
+    if (!this.isTriangle()) {
+      throw new Error('this is not triangle');
+    }
```

Comment:
you can just call `this.isTriangle()` without the if statement - and move the error message to the private method